docs/application/tizen-studio/index.md

```diff
@@ -13,15 +13,15 @@ To develop your applications, use the set of tools provided by the Tizen Studio:
 
 - [Setup](setup/installing-sdk.md) Learn about installing and updating the Tizen Studio. You may need to install additional packages to develop applications for various platforms.
 
-- [Native Tools ](native-tools/cover-native-n.md)Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
+- [Native Tools](native-tools/cover-native.md)Learn about the available native application development tools. Native applications are developed using the C programming language. You can create an application project, edit the code to meet your needs, and run the application on the emulator or a target device.
```

Comment:
Wouldn't it better to have a space between `[Native Tools](native-tools/cover-native.md)` and `Learn`?

Reply:
I'm modifying index.md and TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.

I  will change file names and directory paths under tizen-studio directory.